src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -416,12 +417,24 @@ def partlabel(self):
             return lsblk_partlabel
         return self.device.blkid_api.get('PARTLABEL', '')
 
+    @property
+    def parttype(self):
+        """
+        Seems like older version do not detect PARTTYPE correctly (assuming the
+        info in util/disk.py#lsblk is still valid).
+        SImply resolve to using blkid since lsblk will throw an error if asked
```

Comment:
probably fine for this fix here, but we should look in `lsblk` as to why it raises an error when asked for an unknown column

Reply:
I think this is just how lsblk is implemented.